Description: Add SO version

--- lofarbeam-4.0.orig/CMakeLists.txt
+++ lofarbeam-4.0/CMakeLists.txt
@@ -27,6 +27,10 @@ add_library(stationresponse SHARED
 add_library(elementresponse SHARED
   ElementResponse.cc)
 
+set_target_properties(stationresponse PROPERTIES VERSION 3) 
+set_target_properties(elementresponse PROPERTIES VERSION 3) 
+
+
 install (TARGETS stationresponse DESTINATION lib)
 install (TARGETS elementresponse DESTINATION lib)
 
